### PR TITLE
Getting Started Visual sweep

### PR DESCRIPTION
**Pull request by jeanservaas**
This is a visual sweep (not meant to change written content or alter meaning); everything that I've done it listed in detail in this issue:

https://github.com/carbon-design-system/carbon-website/issues/792

- Still need to update image on Developers > Angular page; but need to get more info first.

### CODE REVIEW

File: src/content/getting-started/designers/designers.md [hunk 1]
```diff
@@ -2,16 +2,38 @@
 title: Designers
 ---
 
-This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+## Resources
+
+<grid-wrapper col_lg="8" flex="true" bleed="true">
+<clickable-tile
+    title="Carbon Design Kit"
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.md]: Change to "Download from GitHub" and change icon to GitHub icon

File: src/content/getting-started/designers/designers.md [hunk 2]
```diff
@@ -2,16 +2,38 @@
...
+    href="https://github.com/ibm/carbon-design-kit"
+    type="resource"
+     >
+    <img src="images/sketch-icon.png" alt="Carbon Design Kit"  />
+</clickable-tile>
+<clickable-tile
+    title="Download Box Drive."
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.md]: Change to "Connect with Box" add a second line saying "IBMers only"

File: src/content/getting-started/designers/designers.md [hunk 3]
```diff
@@ -2,16 +2,38 @@
 title: Designers
 ---
 
-This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon.
+## Resources
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.md]: Can we change to "Get the kit" and the two tiles are the different ways of accessing it.

File: src/content/getting-started/designers/designers.md (hunk 1 above)
Comment (aagonzales) [reply src/content/getting-started/designers/designers.md]: If we don't change the title of the section then we need to say "Download the kit from GitHub"

File: src/content/getting-started/designers/designers.md (hunk 2 above)
Comment (aagonzales) [reply src/content/getting-started/designers/designers.md]: f we don't change the title of the section then we need to say "Connect the kit through Box"

File: src/content/getting-started/designers/designers.md [hunk 4]
```diff
@@ -2,16 +2,38 @@
...
+This site, along with the Carbon Design Kit, provides designers with everything they need to get up and running with Carbon. The Carbon Design Kit is a constantly evolving document that contains all of the design system's visual assets, including components, icons, color palettes, grids, etc. When used properly, the kit should help you work faster, with confidence that your product or experience is well aligned with the IBM Design System. It utilizes symbols and text styles, organized via pages in the left side panel of Sketch.
 
-The design kit is available to the public through GitHub and internally to IBMers through Box Drive. The kit can be used directly in the Sketch application or loaded as a Sketch library source.
+The design kit is available to the public through GitHub and [internally to IBMers through Box Drive.](#synchronize-via-box-drive) The kit can be used directly in the Sketch application or loaded as a Sketch library source.
 
-- [Download from GitHub](#download-from-github)
-- [Use with Box Drive](#synchronize-via-box-drive)
 
 ### Why Sketch?
```
Comment (aagonzales) [inline src/content/getting-started/designers/designers.md]: Lets remove this Section. Jeoff doesn't like it

File: src/content/getting-started/developers/react.md [hunk 5]
```diff
@@ -3,23 +3,37 @@ title: Developers
...
-If you prefer [Yarn](https://yarnpkg.com/en/), use the following command instead:
+####If you prefer [Yarn](https://yarnpkg.com/en/):
+use the following command instead:
 
 ```bash
 $ yarn add carbon-components-react carbon-components carbon-icons
 ```
 
-### Getting started
+### Resources
```
Comment (aagonzales) [inline src/content/getting-started/developers/react.md]: Are there supposed to be resource tiles here?

**Comment by jeanservaas** [comment]
Hey @alisonjoseph and @aagonzales 

I made these changes, you can see them in the commits. Preview link looks like it's still pending.

**Comment by alisonjoseph** [comment]
Yep, that anchor link won't work until my PR is merged in that ignores the special characters.

**Pull request merged**